packages/updater-service/src/repositories/consumerEservice.repository.ts

```diff
@@ -84,9 +84,11 @@ export const consumerEserviceRepository = (
     state: string
   ): Promise<ConsumerEservice> {
     try {
+      const tmstLastEdit = new Date().toISOString();
```

Comment:
For next iterations, I would put it in a function (ex. `getCurrentDate()`)

Reply:
Done :)